src/main/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryController.java

```diff
@@ -387,6 +388,10 @@ public ResponseEntity<InputStreamResource> exportAdvisory(
             return ResponseEntity.internalServerError().build();
         } catch (CsafException ex) {
             return ResponseEntity.status(ex.getRecommendedHttpState()).build();
+        } finally {
+            if (filePath != null) {
+                filePath.toFile().delete();
```

Comment:
This raises the spotbugs warning `RV_RETURN_VALUE_IGNORED_BAD_PRACTICE` because the returned value is not used. I think in this case it's fine to add an ignore for the warning.

In general, a test if the files are really deleted would be nice.